libs/dh/market-participant/data-access-api/src/lib/dh-market-participant-edit-actor-data-access-api.store.ts

```diff
@@ -180,12 +183,16 @@ export class DhMarketParticipantEditActorDataAccessApiStore extends ComponentSto
         actorId
       )
       .pipe(
-        tap((actor) =>
+        tap((actorRes) => {
           this.patchState({
             organizationId,
-            actor,
-          })
-        )
+            actor: {
+              ...actorRes,
+              actorId,
```

Comment:
suggestion: delete this line. The `actionId` should probably come from `actorRes`. I think it was added so I can test locally.